sdk/python/kfp/gcp.py

```diff
@@ -85,3 +87,17 @@ def _set_tpu_spec(task):
         return task
 
     return _set_tpu_spec
+
+def use_preemptible_vm_gpu(toleration: V1Toleration = V1Toleration(effect='NoSchedule',
```

Comment:
I see no mention of GPU in the code.

Reply:
GPU requires special key and value names in the toleration because GKE automatically taint the VM.
Different GPU could lead to different key value name. I'm looking into the different types. Possibly change it.

You could review the PR once WIP is removed.

Reply:
In fact, there is no special handling, the GKE will automatically add another toleration if the pod requires GPU resources.

Reply:
Talked offline:
We've considered several names and chose the last one (`use_preemprible_node_pool`):
* `use_preemprible_vm`
* `use_preemprible`
* `use_preemprible_instance`
* `use_preemprible_node_pool`